.github/workflows/base-package-tests.yaml

```diff
@@ -14,11 +14,15 @@ on:
         type: boolean
         default: false
         description: "Set to true if a GH issue should be generated upon failure"
+      runner_os:
+        required: false
+        type: string
+        default: ubuntu-22.04
```

Comment:
Wanted to bump this to `ubuntu-24.04`, but the upgrade triggers errors in the package tests. These errors be related to systemd. Sadly I don't have enough knowledge to debug and fix them. :/